enterprise/server/cmd/server/main.go

```diff
@@ -60,23 +60,19 @@ func configureFilesystemsOrDie(realEnv *real_environment.RealEnv) {
 	// The static filesystem is the same, however, so we set it if the flag
 	// is set, but we do not fall back to an embedded staticFS.
 	realEnv.SetAppFilesystem(nil)
-	if staticDirectoryFlag := flag.Lookup("static_directory"); staticDirectoryFlag != nil {
-		if staticDirectory := staticDirectoryFlag.Value.String(); staticDirectory != "" {
-			staticFS, err := static.FSFromRelPath(staticDirectory)
-			if err != nil {
-				log.Fatalf("Error getting static FS from relPath: %q: %s", staticDirectory, err)
-			}
-			realEnv.SetStaticFilesystem(staticFS)
+	if staticDirectory, err := flagutil.DereferencedValueFromFlagName[string]("static_directory"); err == nil && staticDirectory != "" {
+		staticFS, err := static.FSFromRelPath(staticDirectory)
+		if err != nil {
```

Comment:
The name is a mouthful, could be something short like flagutil.GetValue()?

Reply:
I think it's important to clarify that it's the dereferenced `Value`, since `flag.Value` is a pointer. How's `GetDereferencedValue` sound?

Reply:
If the caller wants the flag value, in what cases do they care whether it's a pointer or not? 
It's not a big deal, so if you want to call it `GetDereferencedValue` then it's fine.

Reply:
It's mostly only important because `SetValueForFlagName` takes a non-pointer value, and if you don't specify the type (i.e., `T` is `any`), you can't then pass that value to `SetValueForFlagName`, which is the use case we have in `server/util/testing/flags/flags.go`'s `Set` function.